### PR TITLE
feat(index): support `window.RUM_PARAMS` to attach custom URL parameters to collection URL

### PR DESCRIPTION
**Pull request by trieloff**

### CODE REVIEW

**Comment by cziegeler** [comment]
LGTM

**Review by cziegeler — APPROVED** [review]
lgtm

**Review by kptdobe — DISMISSED** [review]

File: src/standalone.js [hunk 1]
```diff
@@ -11,7 +11,15 @@
...
-  ? new URL(document.currentScript.src, window.location.origin).origin : null;
-window.RUM_BASE = window.RUM_BASE || scriptSrc;
-sampleRUM();
+try {
+  const scriptSrc = (document.currentScript && document.currentScript.src)
+    ? new URL(document.currentScript.src, window.location.origin).origin : null;
+  const scriptParams = (document.currentScript && document.currentScript.dataset)
+    ? document.currentScript.dataset : null;
+  window.RUM_BASE = window.RUM_BASE || scriptSrc;
+  window.RUM_PARAMS = window.RUM_PARAMS || scriptParams;
```
[the remark this answers is not in the document]
Comment (kptdobe) [reply src/standalone.js:20]: How are those params handled "server side" ? if they cannot create damages, we can leave it open for now and restrict later if needed.

[the remark this answers is not in the document]
Comment (cziegeler) [reply src/standalone.js:20]: For CS we only need one parameter, "routing" - I assume "server side" we look for specific ones only anyway?

**Review by trieloff — COMMENTED** [review]
LGTM

File: src/standalone.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (chicharr) [reply src/standalone.js:20]: According to @trieloff  we don't need to filter params.

**Review by chicharr — APPROVED** [review]
LGTM

**Pull request merged**